### PR TITLE
Identification package

### PR DESCRIPTION
**Pull request by YuriAlessandro**
This PR encompasses the issues #8, #9 and #7

### CODE REVIEW

File: middleware/src/main/java/identification/ObjectId.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+/**
+ * Defines a valid (globally unique) identifier for each remote object instance, 
+ * which is a 64-bit number value.
+ * 
+ * @author Yuri Alessandro Martins 
+ * @version 1.0
+ */
+public class ObjectId {
+	
+	public UUID objectId;
```
Comment (eltonvs) [inline middleware/src/main/java/identification/ObjectId.java]: Use private attributes instead public

File: middleware/src/main/java/identification/ObjectId.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	
+	/**
+	 * Construct new ObjectId
+	 */
+	public ObjectId() {
+		this.objectId = UUID.randomUUID();
+	}
+
```
Comment (eltonvs) [inline middleware/src/main/java/identification/ObjectId.java]: Create a getter for the UUID's least significant bits

File: middleware/src/main/java/br/ufrn/dimap/middleware/identification/lookup/ILookup.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+ * Implements a Lookup Service as part of the distributed object middleware. 
+ * This service provides an way to bind some name to an remote object absolute 
+ * reference, and other way to find this absolute reference given an name.
+ * 
+ * @author Yuri Alessandro Martins 
+ * @version 1.0
+ * @see AbsoluteObjectReference
+ * @see RemoteError
+ */
+public interface ILookup {
```
Comment (greati) [inline middleware/src/main/java/br/ufrn/dimap/middleware/identification/lookup/ILookup.java]: Name the interface simply as Lookup, not ILookup. I've got a good link about this: https://stackoverflow.com/questions/2814805/java-interfaces-implementation-naming-convention/2814831

Comment (YuriAlessandro) [reply middleware/src/main/java/br/ufrn/dimap/middleware/identification/lookup/ILookup.java]: done

File: middleware/src/main/java/identification/ObjectId.java (hunk 1 above)
Comment (YuriAlessandro) [reply middleware/src/main/java/identification/ObjectId.java]: done

File: middleware/src/main/java/identification/ObjectId.java (hunk 2 above)
Comment (YuriAlessandro) [reply middleware/src/main/java/identification/ObjectId.java]: done

**Pull request merged**